index.html

```diff
@@ -5124,25 +5125,28 @@ <h2 id="AudioWorkletNodeOptions">
                 unsigned long numberOfInputs
               </dt>
               <dd>
-                Same to <a>numberOfInputs</a> in <a>AudioNode</a>.
+                Same to <a>numberOfInputs</a> in <a><code>AudioNode</code></a>.
```

Comment:
We have not figured out what this `options` object should look like, so I'll just leave as it is.